pkg/clients/rds/rds_test.go

```diff
@@ -21,9 +48,252 @@ func TestIsErrorNotFound(t *testing.T) {
 	response["Code"] = ErrCodeInstanceNotFound
 
 	responseContent, _ := json.Marshal(response)
```

Comment:
You may consider to assert that the returned error is `nil`:
```suggestion
	responseContent, err := json.Marshal(response)
	if err != nil {
		t.Fatalf(...
	}
```

Reply:
I don't mind whether err is nil or not as it's not the main logic. If err is not nil, `responseContent` won't pass the check. So can I ignore the check on `err`?